src/main/resources/script_templates/ImageJ2-Scripts/Z_Projection.py

```diff
@@ -0,0 +1,54 @@
+# @ImageJ ij
+# @Dataset data
+# @String(label="Projection Type",choices={"Max","Mean","Median","Min", "StdDev", "Sum"}) proj_type
+ 
```

Comment:
At some point we could consider doing the same kind of thing for the thresholding op.  Make the thresholding string an input type, so user can choose different types of thresholding.  That may be something for the next iteration.